### PR TITLE
Dev dlrm async loss

### PR DESCRIPTION
**Pull request by ShawnXuan**

### CODE REVIEW

File: RecommenderSystems/dlrm/train.py [hunk 1]
```diff
@@ -167,16 +128,22 @@ def __call__(self):
...
+        last_iter, last_time = 0, time.time()
         for _ in range(self.max_iter):
             self.cur_iter += 1
             loss = self.train_one_step()
 
-            loss = tol(loss)
-
-            self.meter_train_iter(loss)
+            if self.cur_iter % self.loss_print_every_n_iter == 0 and self.rank == 0:
+                loss = tol(loss).mean().numpy()
```
Comment (ShawnXuan) [inline RecommenderSystems/dlrm/train.py:137]: loss = tol(loss.mean()).numpy()

Comment (ShawnXuan) [reply RecommenderSystems/dlrm/train.py:137]: 这里往下还需要异步执行吗？

File: RecommenderSystems/dlrm/train.py [hunk 2]
```diff
@@ -167,16 +128,23 @@ def __call__(self):
...
+        last_iter, last_time = 0, time.time()
         for _ in range(self.max_iter):
             self.cur_iter += 1
             loss = self.train_one_step()
 
-            loss = tol(loss)
-
-            self.meter_train_iter(loss)
+            if self.cur_iter % self.loss_print_every_n_iter == 0:
+                loss = loss.numpy()
```
Comment (clackhan) [inline RecommenderSystems/dlrm/train.py:137]: 为什么叫 async loss呢，这里的```loss.numpy()```也是同步的呀

Comment (clackhan) [inline RecommenderSystems/dlrm/train.py:137]: 这里的意思是只打印当前 iter 的numpy，而不是every_n_iter的平均值，这是想要的行为吗

Comment (ShawnXuan) [reply RecommenderSystems/dlrm/train.py:137]: Yes, batch size is big(50000+), mean value of single step is good enough to see if it is converging or not.
